tests/tsl/tsl3/test.cs

```diff
@@ -0,0 +1,268 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Cache;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Trinity;
+using Trinity.Core.Lib;
+using Trinity.Storage;
+using Xunit;
+using Xunit.Sdk;
+
+namespace tsl3
+{
+    #region Fixture
+    public class TestServer : TestServerBase
+    {
+        public override void TestSynWithRspHandler(ReqReader request, RespWriter response)
```

Comment:
TODO: should also test multiple message ids within a type. for example, two syn_with_rsp message ids, to see whether the messages are sent to the right handler